cli/README.md

```diff
@@ -69,3 +70,51 @@ Global Flags:
 -kubeconfig, -c=<string>
 	Path to kubeconfig file
 ```
+
+### consul-k8s uninstall
+This command uninstalls Consul on Kubernetes, while prompting whether to uninstall the release and whether to delete all
+related resources such as PVCs, Secrets, and ServiceAccounts.
+
+Get started with:
+```bash
+consul-k8s uninstall
+```
+
+```
+Usage: kubectl consul uninstall [options]
+
+Uninstall Consul and clean up all data.
+Any data store in Consul will not be recoverable.
+Command Options:
+
+  -auto-approve
+      Skip approval prompt for uninstalling Consul. The default is false.
+
+  -name=<string>
+      Name of the installation. This will be prefixed to resources installed
+      on the cluster.
+
+  -namespace=<string>
+      Namespace for the Consul installation.
+
+  -skip-wipe-data
+      Skip deleting all PVCs, Secrets, and Service Accounts associated with
+      Consul Helm installation without prompting for approval to delete. The
+      default is false.
+
+  -wipe-data
+      Delete all PVCs, Secrets, and Service Accounts associated with Consul
+      Helm installation without prompting for approval to delete. Only use
+      this when persisted data from previous installations is no longer
+      necessary. The default is false.
```

Comment:
If skip-wipe-data is defaulted `false`  (meaning wipe the data)  and wipe-data is defaulted `false` (meaning dont wipe the data) what is the actual default behaviour? I'm assuming that we meant that skip-wipe-data is defaulted to `true`?